.vscode/settings.json

```
@@ -1,7 +1,7 @@
 {
   "editor.tabSize": 2,
   "files.trimTrailingWhitespace": true,
-  "editor.formatOnSave": false,
+  "editor.formatOnSave": true,
```

Ok if we leave this change out just to keep things scoped?
```suggestion
  "editor.formatOnSave": false,
```

---

Fixed!